source/software/module_system_basics.rst

```diff
@@ -0,0 +1,327 @@
+
+::
+
+   $ module load CP2K/2023.1-intel-2023a
+
+.. note::
+
+   Loading modules with explicit versions is considered as best practice. It ensures
+   that your scripts will use the expected version of the software, regardless of
+   newly installed software. Failing to do this may jeopardize the reproducibility
+   of your results!
+
+Modules need not be loaded one by one; two ``load`` sub-commands
+can for example be combined as follows:
+
+::
+
+   $ module load CP2K/2023.1-foss-2023a GROMACS/2023.3-foss-2023a-PLUMED-2.9.0
+
+.. warning::
+
+   Do *not* load modules in your ``.bashrc``, ``.bash_profile`` or ``.profile``,
+   you *will* shoot yourself in the foot at some point.  Consider using
+   :ref:`module collections <collections of modules>` ``restore`` as a command
+   line alternative (so *not* in the shell initialization files either!).
+
+
+Conflicting modules
+~~~~~~~~~~~~~~~~~~~
+
+It is important to note that only modules that are compatible with
+each other should be loaded together. The loaded modules should all
+be associated with either the same toolchain or compatible (sub)toolchains
+(see also https://docs.easybuild.io/common-toolchains/#toolchains_diagram).
+
+For example, once you have loaded one or more modules that use the
+``foss/2023a`` toolchain, all other modules that you load should have been
+installed with the same toolchain or with compatible (sub)toolchains such as
+``GCC/12.3.0`` or ``GCCcore/12.3.0``.
+
+Additionally, two versions of the same software packages can not be loaded
+together. If you e.g. loaded a ``Python/3.11.3-GCCcore-12.3.0`` module, then
+also loading another ``Python`` module (either directly or as a dependency of
+another module) will cause ``Python/3.11.3-GCCcore-12.3.0`` to be unloaded and
+replaced by the new module (the same will happen to the modules which both
+``Python`` modules load as dependencies).
+
+
+List loaded modules
+~~~~~~~~~~~~~~~~~~~
+
+Obviously, the user needs to keep track of the modules that are
+currently loaded. After executing the above load command, the list
+of loaded modules will look similar to:
+
+::
+
+   $ module list
+   Currently Loaded Modulefiles:
+     1) cluster/wice/batch
+     2) GCCcore/10.3.0
+     ...
+     16) OpenMPI/4.1.1-GCC-10.3.0
+     17) OpenBLAS/0.3.15-GCC-10.3.0
+     ...
+     46) PLUMED/2.9.0-foss-2023a
+     47) CP2K/2023.1-foss-2023a
+     48) GROMACS/2023.3-foss-2023a-PLUMED-2.9.0
+
+Note that this does not just show the two requested modules, but also all
+the modules that got loaded automatically in order to satisfy (runtime)
+dependencies of the explicitly loaded ``CP2K`` and ``GROMACS`` installations
+(``PLUMED``, ``OpenMPI``, ``OpenBLAS``, etcetera).
+
+
+Unloading modules
+~~~~~~~~~~~~~~~~~
+
+To unload a module, one can use the ``module unload`` command. It works
+consistently with the ``load`` command, and reverses the latter's
+effect. One can however unload automatically loaded modules manually, to
+debug some problem.
```

Comment:
problems(?).
To the limit of my broken English knowledge, the 'problem' in this context is countable (because one may be able to list a finite number of issues he has), and when prepended with 'some', then the noun 'problem' gets a plural form by an ending 's'. But, please double check.

Reply:
(addressed in c683e78bb9f7b0baa7f59c33d8d3b28595bfed97)